frontend/app/equity/dividend_computations/new/page.tsx

```
@@ -0,0 +1,246 @@
+  const [isLoading, setIsLoading] = useState(false);
+  const [error, setError] = useState("");
+  const router = useRouter();
+
+  const handleSubmit = async (e: React.FormEvent) => {
+    e.preventDefault();
+    setIsLoading(true);
+    setError("");
+
+    try {
+      // Demo environment - simulate creation and redirect
+      await new Promise(resolve => setTimeout(resolve, 1000)); // Simulate API call
```

ESLint

[error] Replace `resolve` with `(resolve)`

(prettier/prettier)